components/connector/internal/gqlschema/schema.graphql

```diff
@@ -0,0 +1,49 @@
+# Certificate
```

Comment:
Could you already think and propose a concept of generic connector APIs. I mean for example JWT too.

Reply:
Sure, will do.